src/Classical/linear_code.jl

```diff
@@ -425,7 +426,30 @@ function random_information_set(C::AbstractLinearCode)
     return pivot_inds
 end
 
-
+#TODO doc
```

Comment:
"""
random_linear_code(F::CTFieldTypes, n::Int, k::Int; rng::AbstractRNG = Random.seed!())

Return a random `[n, k]` linear code over the field `F`.
"""

indent, add a colon before the `rng` to make optional keyword parameter

Reply:
change `field` to `F` for consistency throughout the library